src/USER-MISC/pair_lebedeva.cpp

```diff
@@ -0,0 +1,437 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Zbigniew Koziol 
+   (National Center for Nuclear Research, Poland)
+   e-mail: softquake at gmail dot com
+   Writing this was based on C code of Kolmogorov-Crespi potential 
+   of Jaap Kroes and others.
+  
+   This is potential described in
+   [Lebedeva et al., Physica E, 44(6), 949-954, 2012.]
+------------------------------------------------------------------------- */
+
+#include <math.h>

```

Comment:
These need to be changed to C++ style includes for C-library APIs (`cmath`, `cstdio`, `cstring`, `cstdlib`)